README.md

```diff
@@ -207,6 +207,125 @@ Você também deveria verificar [a nova documentação do TypeScript para descri
 
 <!--END-SECTION:setup-->
 
+<!--START-SECTION:function-components-->
+
 # Seção 2: Primeiros Passos
 
 ## Componente de Função
+
+Podem ser escritos como funções normais que recebem `props` como argumento e retornam um elemento JSX.
+
+```tsx
+type AppProps = { message: string }; /* também se pode usar uma interface */
+const App = ({ message }: AppProps) => <div>{message}</div>;
+```
+
+<details>
+
+<summary><b>Por que `React.FC` é desencorajado? E sobre `React.FunctionComponent` / `React.VoidFunctionComponent`?</b></summary>
+
+Você pode ver isso em muitas bases de código React + TypeScript:
+
+```tsx
+const App: React.FunctionComponent<{ message: string }> = ({ message }) => (
+  <div>{message}</div>
+);
+```
+
+No entanto, o consenso geral hoje é que o uso de `React.FunctionComponent` (ou a abreviação` React.FC`) é [desencorajado] (https://github.com/facebook/create-react-app/pull/8177). Isto é um ponto de vista, é claro, mas se você concorda e deseja remover `React.FC` da sua base de código, você pode usar [este jscodeshift codemod] (https://github.com/gndelia/codemod-replace-react- fc-typescript).
+
+Algumas diferenças da versão de "função normal":
+
+- `React.FunctionComponent` é explícito sobre o tipo de retorno, enquanto a versão normal da função é implícita (ou então precisa de anotações adicionais).
+
+- Fornece verificação de tipos e preenchimento automático para propriedades estáticas como `displayName`,` propTypes` e `defaultProps`.
+
+- Observe que existem alguns problemas conhecidos usando `defaultProps` com` React.FunctionComponent`. Consulte [este problema para obter detalhes] (https://github.com/typescript-cheatsheets/react-typescript-cheatsheet/issues/87). Nós mantemos uma seção `defaultProps` separada para que você também possa consultar.
+
```

Comment:
I think maybe you forgot to add this topic:

- It provides an implicit definition of `children` (see below) - however there are some issues with the implicit `children` type (e.g. [DefinitelyTyped#33006](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33006)), and it might be better to be explicit about components that consume `children`, anyway.

Reply:
Oh, sorry. My bad. I definitively had skipped this part for an unknown reason. ️🤦‍♂️

I have already done it. It will be included in the next commit. :)